node-operators/block-producers/operational-tasks/upgrade-your-node.md

```diff
@@ -0,0 +1,156 @@
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-skylake" \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+=== "AMD Zen3"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-znver3" \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+### Specifying a Version Tag {: #specifying-a-version-tag }
+
+The Docker commands above will automatically fetch the latest release version tag, but if you wanted to specify a [particular version tag](https://hub.docker.com/r/moondancelabs/tanssi/tags){target=\_blank}, you can do so by appending the version tag to the image name. For example, if you wanted to fetch version `v0.5`, rather than specifying the image name as `moondancelabs/tanssi`, you would indicate `moondancelabs/tanssi:v0.5` as the image name.
+
+The complete commands with specific version tags are thus as follows:
+
+=== "Generic"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    moondancelabs/tanssi:v0.5 \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+=== "Intel Skylake"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-skylake" \
+    moondancelabs/tanssi:v0.5 \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+=== "AMD Zen3"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-znver3" \
+    moondancelabs/tanssi:v0.5 \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+## Upgrading Systemd Nodes {: #upgrading-systemd-nodes }
+
+If you're running your block producer node via the Systemd service, you'll need to take a few steps to properly upgrade your node. In short, you'll need to stop the service, replace the Tanssi binary with the updated version, and then start the service. 
+
+You can stop your Tanssi Systemd service with the following command:
+
+```bash
+systemctl stop tanssi.service
+```
+
+Then, navigate to the directory where your Tanssi binary is stored and remove it. 
+
+```bash
+cd /var/lib/tanssi-data
+```
+
+Note, depending on the particular binary you have, your tanssi node may be named `tanssi-node`, `tanssi-node-skylake`, or `tanssi-node-zen3`.
+
+```bash
+rm tanssi-node
+```
+
+To download the latest release and change permissions on it so the Tanssi service can use it, run the following command that corresponds to your environment:
```

Comment:
These commands changed slightly to use the `-O` flag instead of `mv ./tanssi-node-* ./tanssi-node`

please check out the latest changes on the systemd page!